tests/gitlab/get_mfem_uberenv.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I know these are just a simple scripts, but please add the copyright header as in the scripts insider `../scripts`. Also, consider dropping the `.sh` for uniformity.

Reply:
Done